PsychPython/audio.py

```diff
@@ -249,10 +257,10 @@ def __init__(self, stream, data=None):
             self.handle = PsychPortAudio('FillBuffer', self.stream.handle,
                                          data)
 
-    def fill_buffer(self, data, streaming=0, startIndex='append'):
+    def fill_buffer(self, data, streaming=0, start_index='Append'):
```

Comment:
PsychPortAudio's 'startIndex" parameter does not recognize text strings. Iirc the help text says that omitting the parameter, or setting it to default [] or None would select "append" iff streamingrefill (aka your 'streaming' is non-zero).